src/main/scala/is/hail/variant/Genotype.scala

```diff
@@ -821,6 +828,11 @@ object Genotype {
       g <- Gen.oneOfGen(genExtreme(v), genRealistic(v)))
       yield (v, g)
 
+  def genNonmissingValue: Gen[Genotype] =
+    for (v <- Variant.gen;
+      g <- Gen.oneOfGen(genExtremeNonmissing(v), genRealisticNonmissing(v)))
+      yield g
```

Comment:
I know this is mimicking the code below, but I find this easier to read as:

```scala
def genNonmissingValue: Gen[Genotype] =
  Variant.gen.flatMap(v =>
    Gen.oneOfGen(genExtremeNonmissing(v), genRealisticNonmissing(v))
```
what do you think of this? Is it harder to parse?